core/storage/base_model/gae_models.py

```diff
@@ -1223,8 +1223,9 @@ class BaseSnapshotContentModel(BaseModel):
 
     @staticmethod
     def get_deletion_policy():
-        """Collection rights are deleted only if the corresponding collection
-        is not public.
+        """The content models do not contain any user ID fields, they might be
```

Comment:
What's an example of the "hidden inside the content field" case?

Reply:
`content = {"owner_ids": ["uid_xxx"]}`

Reply:
Oh, OK. Try to explain it more clearly in the docstring so that it's clear what you mean.

Reply:
Done.